framework/src/timesteppers/FunctionDT.C

```diff
@@ -20,7 +20,7 @@ template<>
 InputParameters validParams<FunctionDT>()
 {
   InputParameters params = validParams<TimeStepper>();
-  params.addParam<std::vector<Real> >("time_t", "The values of t");
+  params.addRequiredParam<std::vector<Real> >("time_t", "The values of t");
   params.addParam<std::vector<Real> >("time_dt", "The values of dt");
```

Comment:
Is there a use case where `time_dt` in not needed? If not, then it should be required as well.